docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -428,7 +394,16 @@ endif::[]
 EOF
 ----
 
-* Optionally enable neutron-sriov-nic-agent in the OpenStackDataPlaneNodeSet CR
+. Ensure that the ovn-controller settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as were set in the Compute nodes before adoption. This configuration is stored in the "external_ids" column in the "Open_vSwitch" table in ovsdb:
```

Comment:
(Not sure if this is in scope of this patch, but I see you do backticks elsewhere.) Maybe `ovn-controller` (with backticks) and `external_ids` and `Open_vSwitch` and "ovsdb -> Open vSwitch database"?